addons/website_livechat/static/src/public_models/public_livechat_global.js

```diff
@@ -31,4 +31,11 @@ registerPatch({
             return this._super();
         },
     },
+    fields: {
+        hasWebsiteLivechatFeature: {
+            compute() {
+                return true;
```

Comment:
I think we will likely drop support of `default` (and `related`), and generalise everything with `compute`.

The only reason we use `default` and `related` is because it's more succinct. However they scale poorly to change, to the point that they quickly need to be turned into `compute` when functional needs become more complex.

So IMO it's not worth introducing the support.

Reply:
I'm not sure I agree with that, I think this kind of "syntactic sugar" is fine and better convey the intent. As for the scalability issue, it does not sound like a big deal turning them into compute once some functional requirements need it. I also feel like this might be unnecessary burden on the update cycle.

That being said, okay to keep it as computed fields for now, as the feature for patching defaults does not exit yet and I don't really want to afford delaying the merge of this commit any further.

Reply:
The feature of `default` has good ideas, but implementation and API is lacking in a way that is not scalable.

For the time being, it's best to keep `default` while not adding non-that-useful features on top of it.
Really `default` just helps to be a bit more succinct and DRY. It shows quickly limitations and forces us to use `compute` (which is actually a good thing, in that this feature is much better designed).